installer/roles/kubernetes/vars/kubernetes.yml

```diff
@@ -0,0 +1,2 @@
+---
+uwsgi_bash: "bash -c"
```

Comment:
Can you configure your editor to ensure newlines on save?

Reply:
will do.  I think I have it fixed now.  Please let me know if you see any more and I'll happily clean that up.